dbt/models/model/docs.md

```diff
@@ -113,7 +113,7 @@ The stages are:
 Identical to `model.performance`, but additionally broken out by quantile.
 
 **Primary Key**: `year`, `run_id`, `stage`, `triad_code`, `geography_type`,
-`geography_id`, `by_class`, `quantile`
```

Comment:
This makes the model performance unique. I don't know if it should work that way or not.